Magento_Catalog/styles/modules/_fotorama.scss

```diff
@@ -0,0 +1,93 @@
+.fotorama {
+    & .fotorama__nav-wrap {
```

Comment:
`&` I don't like this ampersand. Please remove it, As I can see we didn't use it so far in alpaca. Please change it in whole file.

Reply:
ok, no problem